engine/apps/schedules/tasks/refresh_ical_files.py

```diff
@@ -2,7 +2,7 @@
 
 from apps.alerts.tasks import notify_ical_schedule_shift
 from apps.schedules.ical_utils import is_icals_equal
-from apps.schedules.tasks import notify_about_empty_shifts_in_schedule, notify_about_gaps_in_schedule
```

Comment:
There _was_ a naming clash between _modules_ and _functions_ with the same name here. This ☝️ `import` statement is importing the _modules_ `notify_about_empty_shifts_in_schedule` and `notify_about_gaps_in_schedule` from the `engine/apps/schedules/tasks` directory _not_ the similarly named functions that're imported into `engine/apps/schedules/tasks/__init__.py`.